jquery.selector-set.next.js

```diff
@@ -30,7 +30,7 @@ function selectorSetHandlers(event) {
     if (matches.length) {
       handlerQueue.push({elem: cur, handlers: matches});
     }
-  } while (cur = cur.parentElement);
```

Comment:
There shouldn't be, since TextNode can't contain element nodes. It's impossible to traverse up using `parentNode` and end up with anything other than Element, Document, or DocumentFragment nodes. The latter are handled by `nodeType !== 1` check above.